lib/rubocop/cop/rspec/rails/inferred_spec_type.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module Rails
+        # Identifies redundant spec type.
```

Comment:
Should we add a little more information about `infer_spec_type_from_file_location!` here? I think the `@safety` notice is not enough on its own.

Reply:
@r7kamura do you have time to address this last note?